src/commands.ts

```diff
@@ -312,7 +312,7 @@ export function registerCommands(
 			} catch (e) {
 				if (e.gitErrorCode === GitErrorCodes.BranchNotFullyMerged) {
 					const action = await vscode.window.showErrorMessage(
-						vscode.l10n.t('The local branch \'{0}\' is not fully merged. Are you sure you want to delete it?', pullRequestModel.localBranchName),
+						vscode.l10n.t('The local branch \'{0}\' is not fully merged. Are you sure you want to delete it?', [pullRequestModel.localBranchName]),
```

Comment:
This isn't right. It should work with rest arguments. If it doesn't, that's a bug in localization

Reply:
Is localBranchName a string?